scripts/common_test.py

```diff
@@ -350,8 +350,8 @@ def _redirect_stdout(new_target):
     def test_install_npm_library(self):
 
         def _mock_subprocess_call(unused_command):
-            """Mocks subprocess.call() to create a temporary file instead of the
-            actual npm library.
+            """Mocks subprocess.call() to create a temporary file instead
+            of the actual npm library.
```

Comment:
Why this change?

Reply:
This was a remnant of a code change that I undone. I've reverted this change.